src/Libraries/Microsoft.Extensions.AI.Evaluation.Reporting/Directory.Build.props

```diff
@@ -0,0 +1,8 @@
+<Project>
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
+
+  <PropertyGroup>
+    <!-- Enable this property during development and CI so that the report is built and embedded in the assembly. -->
+    <!-- <EnableEvaluationReportBuild>true</EnableEvaluationReportBuild> -->
```

Comment:
I'd rather that we use an env var and delete this file to avoid accidentally committing it.

Reply:
Setting `EnvEnableEvaluationReportBuild` will now set this property and enable the report build.